src/worker/core/trade/filterItem.ts

```diff
@@ -0,0 +1,19 @@
+export default class FilterItem {
+	filterData: any; //filter items selected, ex for Pos: C, PF...
```

Comment:
Don't use `any` here, or in any (ha) of the other functions in this class. I know you have some cases where this is an array and others where it's a string, but that is a classic example of when a generic should be used.

Reply:
you're right! like I said, still new TS so I appreciate you bearing with me as I learn